packages/flutter/lib/src/material/scrollbar.dart

```diff
@@ -15,149 +13,67 @@ const Duration _kScrollbarTimeToFade = Duration(milliseconds: 600);
 
 /// A material design scrollbar.
 ///
-/// A scrollbar indicates which portion of a [Scrollable] widget is actually
-/// visible.
+/// To add a scrollbar thumb to a [ScrollView], simply wrap the scroll view
+/// widget in a [Scrollbar] widget.
+///
+/// {@macro flutter.widgets.Scrollbar}
 ///
-/// Dynamically changes to an iOS style scrollbar that looks like
-/// [CupertinoScrollbar] on the iOS platform.
 ///
-/// To add a scrollbar to a [ScrollView], simply wrap the scroll view widget in
-/// a [Scrollbar] widget.
+// TODO(Piinks): Add code sample
 ///
 /// See also:
 ///
+///  * [RawScrollbar], the simple base class this extends.
+///  * [CupertinoScrollbar], an iOS style scrollbar.
 ///  * [ListView], which display a linear, scrollable list of children.
 ///  * [GridView], which display a 2 dimensional, scrollable array of children.
-class Scrollbar extends StatefulWidget {
+class Scrollbar extends RawScrollbar {
   /// Creates a material design scrollbar that wraps the given [child].
   ///
   /// The [child] should be a source of [ScrollNotification] notifications,
   /// typically a [Scrollable] widget.
```

Comment:
Shouldn't this comment talk about the other arguments too? (mainly I'm thinking about what it means to not specify the optional parameters).

Reply:
Added! :)